spec/index.bs

```diff
@@ -2066,11 +2069,13 @@ namespace GPUBufferUsage {
                     - If |descriptor|.{{GPUBufferDescriptor/mappedAtCreation}} is `true`:
                         - |descriptor|.{{GPUBufferDescriptor/size}} is a multiple of 4.
 
-                    Issue(gpuweb/gpuweb#605): Explain that the resulting error buffer can still be mapped at creation.
-
                     Issue(gpuweb/gpuweb#605): Explain what are a {{GPUDevice}}'s `[[allowed buffer usages]]`.
                 </div>
 
+            Note: If buffer creation fails, and |descriptor|.{{GPUBufferDescriptor/mappedAtCreation}} is `false`,
```

Comment:
I don't quite understand who is this note targeted to.
Does it inform the user in any actionable way?
Does it inform the implementors?

Reply:
I think it informs both:
Users know they don't have to do anything special to free shadow resources for failed buffers.
Implementers know that's when they should free shadow copies/etc made for mapping.

Reply:
> Users know they don't have to do anything special to free shadow resources for failed buffers.

I don't think anybody would assume that anything has to be done in the first place.

Reply:
Maybe not at first, but a user who is a little bit familiar with the implementation architecture (which is a good chunk of users, usually) may well be unsure. In any case, I think it still informs implementers, so shouldn't we keep it for that reason?

Reply:
This implementation behavior is non-observable, and I don't even see this as something our implementation would follow.

Reply:
How do you even `destroy()` a failed buffer? A failed buffer shouldn't support any operations on it, anyway.

Reply:
I don't think destroying a failed buffer should do anything, but I also think we should allow it so applications don't have to special-case their cleanup logic.